Modules/Core/Common/test/itkBuildInformationGTest.cxx

```diff
@@ -49,10 +49,15 @@ TEST(ITKBuildInformation, InformationFeatures)
 
   for (auto mapEntry : localMap)
   {
+    const auto key = mapEntry.first;
+
```

Comment:
Why not add `value` as a local variable `auto value = mapEntry.second;`? This discrepancy triggers my OCD 😄

Reply:
> Why not add value as a local variable

@dzenanz Thanks for the suggestions, but the thing that bothers me is the inability to pass an l-value `key` as argument to `GetValue` and `GetDescription`. So  `key` is the key thing of this pull request  😸